config/application.rb

```diff
@@ -6,7 +6,7 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
-module Hideout
+module HideoutWeb
```

Comment:
Any reason for this change?

Reply:
Caused conflicts with the Hideout Active Record class. 
For example, Hideout.new would not work because Hideout mapped to the module.

Reply:
Got it